community/examples/fsi-montecarlo-on-batch.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: fsi-montecarlo-on-batch
+
+vars:
+  project_id: <my_project>
+  deployment_name: fsimontecarlo
+  region: us-central1
+  zone: us-central1-a
+
+deployment_groups:
+- group: setup
+  modules:
+  - id: enable_apis
+    source: community/modules/project/service-enablement
+    settings:
+      gcp_service_list: [
+        "bigquery.googleapis.com",
+        "container.googleapis.com",
+        "logging.googleapis.com",
+        "notebooks.googleapis.com",
+        "batch.googleapis.com",
+        "pubsub.googleapis.com",
+        "compute.googleapis.com"
+      ]
+
+- group: primary
+  modules:
+
+  - id: new-notebook
+    source: ./community/modules/compute/notebook
```

Comment:
Use embedded modules. AKA remove the `./` from all the source lines.

Reply:
Yep.